inductiva/storage/storage.py

```diff
@@ -0,0 +1,89 @@
+"""Methods to interact with the user storage resources."""
+import inductiva
+from inductiva.client.apis.tags import instance_api
+from inductiva.utils import format_utils
+from typing import Literal
+
+
+def get_space_used():
+    """Returns the occupied storage size in GB."""
+    try:
+        api = instance_api.InstanceApi(inductiva.api.get_client())
+        response = api.get_storage_size()
+        storage_used = round(float(response.body), 3)
+        print(f"Total storage used: {storage_used} GB")
+        return storage_used
+    except inductiva.client.ApiException as api_exception:
+        raise api_exception
+
+
+def listdir(max_results: int = 3,
+            order_by: Literal["size", "creation_time"] = "size",
+            sort_order: Literal["asc", "desc"] = "desc"):
+    """List and display the contents of the user's storage bucket.
+    Args:
+        max_results (int): The maximum number of results to return.
+        order_by (str): The field to sort the contents by.
+        sort_order (str): Whether to sort the contents in ascending or
+        descending order.
+    Returns:
+        list of dict: A list of dictionaries containing information about 
+        the size, the name and the creation time of each content that can
+        easily be converted to a dataframe. 
+
+    This function prints a table with the storage content information: 
+        Name            Size            Creation Time
+        1234            5.68 MiB        29 Sep, 14:12:00
+        12345           374.85 KiB      29 Sep, 14:13:10
+        1234567         97.59 KiB       29 Sep, 14:13:24
+        123             0 B             29 Sep, 14:13:29
+        123456          0 B             29 Sep, 14:13:18
+    """
+    try:
+        api = instance_api.InstanceApi(inductiva.api.get_client())
+        contents = api.list_storage_contents({
+            "max_results": max_results,
+            "sort_by": order_by,
+            "order": sort_order
+        }).body
+        all_contents = []
+        for content_name, info in contents.items():
+            size = info["size_bytes"]
+            creation_time = info["creation_time"]
+            all_contents.append({
+                "content_name": content_name,
+                "size": round(float(size), 3),
+                "creation_time": creation_time
+            })
+        print(_print_contents_table(all_contents))
```

Comment:
You don't have to answer here but should printing be optional / contextual?
I think printing/logging/returning is something that needs to be discussed at a much higher level, but please bear in mind that it's a design choice we will need to discuss with @hpenedones